examples/nlp/dialogue_state_tracking/data/multiwoz/create_data_from_multiwoz.py

```diff
@@ -0,0 +1,793 @@
+# =============================================================================
+# Copyright 2020 NVIDIA. All Rights Reserved.
+# Copyright 2019 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+"""Converts Multiwoz 2.1 dataset to the data format of SGD."""
```

Comment:
Just a question: should we indicate in any way which code artifacts/functions come from Google, and which code is ours?